### ISSUE TITLE
Facebook SSO onActivityResult / Facebook Callback for user choice not called if using Chrome Tabs -- hanging waiting for result

### ISSUE THREAD

**Issue by laurenyew-nytimes** [facebook/facebook-android-sdk#1107]
### Checklist before submitting a bug report

- [X] I've updated to the latest released version of the SDK
- [X] I've searched for existing [Github issues](https://github.com/facebook/facebook-android-sdk/issues)
- [X] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [X] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [X] This issue is not security related and can safely be disclosed publicly on GitHub

### Java version

11

### Android version

API 33

### Android SDK version

14.1.1, 12.1.0, 13.1.0

### Installation platform & version

Gradle 7.2.1

### Package

Login

### Goals

I have several different apps with Facebook SSO working with Chrome tabs. I'm trying to set up Facebook SSO on my newest app. Currently Facebook SSO is not working on my new app if it opens up using Chrome Tabs.
I would like Facebook SSO to be able to login even if don't have Facebook app installed on the phone (using Chrome Tabs).
Currently Facebook SSO works when I have Facebook app installed on the phone, but when I don't and use Chrome Tabs, the `onActivityResult` is never getting called even though the Chrome Tabs are closing with the choice.
I noted that the app is using the Facebook app id for another app + packageName but I don't know if that's the issue because Facebook SSO is working if Facebook is installed on the phone.

### Expected results

Facebook SSO `onActivityResult` should be called with the result

### Actual results

`onActivityResult` is not called until I press a back button twice, otherwise I just hang out waiting on a result

### Steps to reproduce

1. Don't have facebook installed on the phone
2. Open up Facebook SSO with chrome tabs from my Android app
3. Make a choice (cancel or approve)
4. See the Facebook SSO chrome tab go away, taking me back to my last screen which is hanging waiting for a result.

### Code samples & details

```java
// INSERT YOUR CODE HERE
class FacebookSSOProviderImpl(
    private val facebookSignIn: FacebookSignInWrapper = FacebookSignInWrapper(),
) : FacebookCallback<LoginResult> {

    private val callbackManager: CallbackManager = CallbackManager.Factory.create()
   
    init {
        facebookSignIn.registerCallback(callbackManager, this)
    }

    override suspend fun ssoLogin(activity: FragmentActivity, accountName: String?): SSOResult {
        val ssoFragment = attachSSOFragment(activity, this, ssoFragmentBuilder)
        facebookSignIn.logInWithReadPermissions(ssoFragment, FACEBOOK_PERMISSIONS)
        return authCodeFlow.first()
    }

    override fun onActivityResult(fragment: SSOFragment, requestCode: Int, resultCode: Int, data: Intent?) {
        facebookSignIn.onActivityResult(callbackManager, requestCode, resultCode, data)
        removeSSOFragment(fragment)
    }

....
/**
 * Wrapper for the Facebook signin library. Used to facilitate testing.
 */
class FacebookSignInWrapper {
    fun registerCallback(callbackManager: CallbackManager, callback: FacebookCallback<LoginResult>) =
        LoginManager.getInstance().registerCallback(callbackManager, callback)

    fun logOut() = LoginManager.getInstance().logOut()

    fun logInWithReadPermissions(ssoFragment: SSOFragment, permissions: Set<String>) =
        LoginManager.getInstance().logInWithReadPermissions(ssoFragment, permissions)

    fun onActivityResult(callbackManager: CallbackManager, requestCode: Int, resultCode: Int, data: Intent?) =
        callbackManager.onActivityResult(requestCode, resultCode, data)
}
```

**Comment by ahulyk** [comment]
Can confirm same issue SDK "12.3.0"

**Comment by 2shrestha22** [comment]
Any workaround for this issue?

**Comment by okq91** [comment]
@laurenyew-nytimes I'm facing the same issue here with custom tabs, when the Facebook App isn't installed on the device onActivityResult is never getting called.
Have you found any workaround?
Facebook login SDK version
`implementation 'com.facebook.android:facebook-login:14.1.0'
`

**Comment by 2shrestha22** [comment]
@laurenyew-nytimes @okq91 here is the solution: https://github.com/facebook/facebook-android-sdk/issues/1095#issuecomment-1233624901
Solved by using: `implementation 'com.facebook.android:facebook-login:14.1.1'`

**Comment by okq91** [comment]
@laurenyew-nytimes Thank you it works fine now using version 14.1.1.

**Comment by laurenyew-nytimes** [comment]
Conformed this is fixed on 14.1.1 version. Thanks all!
